.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "cinch"]
+	path = cinch
+	url = ssh://xcp-stash.lanl.gov:7999/laristra/cinch
```

Comment:
This should be the GitHub repo.

Reply:
That's correct. Good catch ! Thanks !

Reply:
Actually what you really want here is something like `../cinch`, so that this doesn't have to change every time you push to a new repo.  I don't remember the exact syntax offhand, but I believe Portage already does this so you can copy from there.